src/app/app.module.js

```diff
@@ -13,4 +13,21 @@ export default angular.module('app', [uiRouter, angularanimate, angulararia, ang
     .config(routes)
     .component('hello', helloComponent)
     .component('world', worldComponent)
-    .component('app', appComponent);
\ No newline at end of file
+    .component('app', appComponent)
+    .factory('weatherService', ['$http', function ($http) {
+        const URL = 'http://api.openweathermap.org/data/2.5/weather';
+    
+        const request = {
+            method: 'GET',
+            url: URL,
+            params: {
+                q: 'Vigo',
+                mode: 'json',
+                units: 'metric',
+                lang: 'en',
+                appid: '37499dd44886aa89565abd094ecbfc8c'
```

Comment:
This repository is public, isn't? :) You must not commit such things as passwords or apikeys into public repositories. You can take a look at https://webpack.js.org/plugins/define-plugin/